Updated documentation for various components.

mattiasberlamont opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Screenshots (if appropriate):

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the styleguide CHANGELOG accordingly.

mattiasberlamont committed:
Updated Contact Details documentation.

Small improvement in wording.

mattiasberlamont committed:
Updated Theme Footer documentation.

Added a section about the contact information in the theme footer, what can be and what cannot be done.

mattiasberlamont committed:
Updated Quote documentation.

Small fix.

mattiasberlamont committed:
Updated Highlight documentation.

mattiasberlamont committed:
Updated Highlight documentation.

mattiasberlamont committed:
Updated Highlight documentation.

Small fixes. Correct link to Call-to-action.

mattiasberlamont committed:
Updated Call to action documentation.

mattiasberlamont committed:
Added Textarea documentation.

delrueba commented on components/31-molecules/cta-block/README.md:
```diff
@@ -1,17 +1,49 @@
+* A link to an e-mail address (mailto-link)
+* A link to a telephone number (tel-link)
+
+The optional secondary action can be one of the following types:
```
This doesn't match the designs of the LEZ tool.
There, the second action is an email.

If you agree, I'd also add a limitation that they can't put links in the text part either.
I've noticed attempts to use this as workaround to have more than two actions.

mattiasberlamont replied:
1/ The secondary action could be indeed also an e-mail address or a telephone number. I cannot see why we would not allow that in the Style Guide.

The only problem I can think of is that one could show two e-mail address or two telephone number without a clear difference. This can be confusing for users. What do you think?

2/ Also a good idea to mention that in the text there cannot be links.

mattiasberlamont replied:
@delrueba The documentation has been updated.

delrueba replied:
1/ It's not included in the allowed types in your list ;)

2/ :+1: 

3/ I like this list style :)

hstandaert commented on components/31-molecules/cta-block/README.md:
```diff
@@ -1,17 +1,49 @@
 
-Examples of paragraphs are defined in paragraph variants:
+* To guide users to the next step to reach a certain goal related to the page.
+* To present specific contact information to get help or to get more information from a specitic service to reach a certain goal.
```
```suggestion
* To present specific contact information to get help or to get more information from a specific service to reach a certain goal.
```

hstandaert commented on components/31-molecules/cta-block/README.md:
```diff
@@ -1,17 +1,49 @@
+* A link to a web page (internal or external)
+* A link to a document
+
+Links to a web pages, e-mail addresses and telephone numbers are presented using a <a href="{{path './link.html'}}">link</a>.
```
```suggestion
Links to web pages, e-mail addresses and telephone numbers are presented using a <a href="{{path './link.html'}}">link</a>.
```

mattiasberlamont committed:
Update components/31-molecules/cta-block/README.md

Co-Authored-By: Helena Standaert <helena.standaert@gmail.com>

mattiasberlamont committed:
Update components/31-molecules/cta-block/README.md

Co-Authored-By: Helena Standaert <helena.standaert@gmail.com>

mattiasberlamont committed:
Updated Call to action documentation.

Added more information about which types of actions to use and added limitations for adding links in the text of a a call to action. Added a small section with things to note when using a call to action.

mattiasberlamont commented:
Can this one be merged?
